x-pack/plugins/security_solution/cypress/e2e/exceptions/shared_exception_lists_management/manage_exceptions.cy.ts

```diff
@@ -0,0 +1,165 @@
+  before(() => {
+    esArchiverResetKibana();
+    esArchiverLoad('exceptions');
+    login();
+    createRule(getNewRule());
+  });
+
+  beforeEach(() => {
+    visitWithoutDateRange(EXCEPTIONS_URL);
+    waitForExceptionsTableToBeLoaded();
+  });
+  after(() => {
+    esArchiverUnload('exceptions');
+  });
+
+  afterEach(() => {
+    deleteCustomRule();
+  });
+
+  const exceptionName = 'My item name';
+  const exceptionNameEdited = 'My item name edited';
+  const FIELD_DIFFERENT_FROM_EXISTING_ITEM_FIELD = 'agent.name';
+  const EXCEPTION_LIST_NAME = 'Newly created list';
+
+  describe('Add, Edit and delete Exception item', () => {
+    it('should create exception item from Shared Exception List page and linked to a Rule', () => {
+      // Click on "Create shared exception list" button on the header
+      cy.get(MANAGE_EXCEPTION_CREATE_BUTTON_MENU).click();
+      // Click on "Create exception item"
+      cy.get(MANAGE_EXCEPTION_CREATE_BUTTON_EXCEPTION).click();
+
+      // Add exception item name
+      addExceptionFlyoutItemName(exceptionName);
+
+      // Add Condition
+      editException(FIELD_DIFFERENT_FROM_EXISTING_ITEM_FIELD, 0, 0);
+
+      // Confirm button should disabled until a rule(s) is selected
+      cy.get(CONFIRM_BTN).should('have.attr', 'disabled');
+
+      // select rule
+      cy.get(RULE_ACTION_LINK_RULE_SWITCH).find('button').click();
+
+      // should  be able to submit
+      cy.get(CONFIRM_BTN).should('not.have.attr', 'disabled');
+
+      submitNewExceptionItem();
+
+      // Navigate to Rule page
+      visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+      goToRuleDetails();
+
+      goToExceptionsTab();
+
+      // Only one Exception should generated
+      cy.get(EXCEPTION_ITEM_VIEWER_CONTAINER).should('have.length', 1);
+
+      // validate the And operator is displayed correctly
+      cy.get(EXCEPTION_CARD_ITEM_NAME).should('have.text', exceptionName);
+    });
+
+    it('should create exception item from Shared Exception List page, linked to a Shared List and validate Edit/delete in list detail page', function () {
+      createSharedExceptionList(
+        { name: 'Newly created list', description: 'This is my list.' },
+        true
+      );
+
+      // After creation - directed to list detail page
+      cy.get(EXCEPTIONS_LIST_MANAGEMENT_NAME).should('have.text', EXCEPTION_LIST_NAME);
+
+      // Go back to Shared Exception List
+      visitWithoutDateRange(EXCEPTIONS_URL);
+
+      // Click on "Create shared exception list" button on the header
+      cy.get(MANAGE_EXCEPTION_CREATE_BUTTON_MENU).click();
+      // Click on "Create exception item"
+      cy.get(MANAGE_EXCEPTION_CREATE_BUTTON_EXCEPTION).click();
+
+      // Add exception item name
+      addExceptionFlyoutItemName(exceptionName);
+
+      // Add Condition
+      editException(FIELD_DIFFERENT_FROM_EXISTING_ITEM_FIELD, 0, 0);
+
+      // select rule
+      cy.get(LINK_TO_SHARED_LIST_RADIO).click();
```

Comment:
NIT: Please wrap this and below line of code into a task method.

Reply:
Done please check [e0e24f7](https://github.com/elastic/kibana/pull/152757/commits/e0e24f779c6479f762916494d5b49264f8886cf6)